subiquity/server/controllers/tests/test_filesystem.py

```diff
@@ -44,3 +49,54 @@ def test_probe_defaults_and_os(self):
         self.app.opts.use_os_prober = True
         run_coro(self.fsc._probe_once(context=None, restricted=False))
         self.app.prober.get_storage.assert_called_with({'defaults', 'os'})
+
+
+class TestGuided(TestCase):
+    def _guided_direct(self, bootloader, ptable):
+        self.app = make_app()
```

Comment:
Wait, doesn't guided always get you a gpt partition table?

Reply:
Yes.  I was working on things related to msdos tables after this and squished that commit into here.  So for now this code is unit-test only.